letta/constants.py

```diff
@@ -46,6 +46,12 @@
     "archival_memory_search",
 ]
 
+# The name of the tool used to send message to the user
+# May not be relevant in cases where the agent has multiple ways to message to user (send_imessage, send_discord_mesasge, ...)
+# or in cases where the agent has no concept of messaging a user (e.g. a workflow agent)
+DEFAULT_MESSAGE_TOOL = "send_message"
```

Comment:
This is basically what needs to be overridden to have `AssistanceMessage` parsed from `send_message_custom` right?